--- a/src/lj_dispatch.c	2012-01-26 00:50:32.494836382 +0400
+++ b/src/lj_dispatch.c	2012-02-04 18:48:01.061810534 +0400
@@ -33,7 +33,7 @@
 
 /* -- Dispatch table management ------------------------------------------- */
 
-#if LJ_TARGET_MIPS
+/*#if LJ_TARGET_MIPS
 #include <math.h>
 LJ_FUNCA_NORET void LJ_FASTCALL lj_ffh_coroutine_wrap_err(lua_State *L,
 							  lua_State *co);
@@ -43,7 +43,7 @@
   GOTDEF(GOTFUNC)
 };
 #undef GOTFUNC
-#endif
+#endif*/
 
 /* Initialize instruction dispatch table and hot counters. */
 void lj_dispatch_init(GG_State *GG)
@@ -63,9 +63,9 @@
   GG->g.bc_cfunc_ext = GG->g.bc_cfunc_int = BCINS_AD(BC_FUNCC, LUA_MINSTACK, 0);
   for (i = 0; i < GG_NUM_ASMFF; i++)
     GG->bcff[i] = BCINS_AD(BC__MAX+i, 0, 0);
-#if LJ_TARGET_MIPS
+/*#if LJ_TARGET_MIPS
   memcpy(GG->got, dispatch_got, LJ_GOT__MAX*4);
-#endif
+#endif*/
 }
 
 #if LJ_HASJIT
